sstables_loader.cc

```diff
@@ -263,6 +263,9 @@ future<> tablet_sstable_streamer::stream() {
 }
 
 future<> sstable_streamer::stream_sstables(const dht::partition_range& pr, std::vector<sstables::shared_sstable> sstables) {
+    size_t nr_sst_total = _sstables.size();
```

Comment:
It looks like `sstables`, not `_sstables` should be used here.
In particular:

```c++
future<> sstable_streamer::stream(std::function<void(unsigned)> on_streamed) {
    const auto full_partition_range = dht::partition_range::make_open_ended_both_sides();

    co_await stream_sstables(full_partition_range, std::move(_sstables), std::move(on_streamed));
}
```

the `_sstables` up above would be empty

Reply:
thanks for pointing this out. indeed.